library/src/test/java/com/datatorrent/lib/io/jms/JMSTransactionableStoreTestBase.java

```diff
@@ -109,6 +112,33 @@ private void createOperator()
     outputOperator.setup(testOperatorContext);
   }
 
+  private JMSStringSinglePortOutputOperator createOperator2()
```

Comment:
Can you eliminate code duplication here? Lines 125-137 match 100-112 (with the exception of outputOperator id which in one case is local and in the other class member and store vs store2 etc). Try to have only one method "createOperator" that deals with outputOperator,store vs outputOperator2,store2 or at least eliminate duplication and remove outputOperator name confusion